src/main/java/com/google/javascript/gents/TypeAnnotationPass.java

```diff
@@ -225,6 +230,13 @@ public void visit(NodeTraversal t, Node n, Node parent) {
           break;
       }
     }
+
+    private void maybeSetTypeExpression(
+        Node nodeToType, JSDocInfo jsDocInfo, boolean isReturnType) {
+      if (jsDocInfo != null) {
```

Comment:
It seems like most callsites do not need the nullability guard in this function. I think I'd prefer to repeat the `!= null` check if it's only two or three sites, it's a bit more explicit in what's going on.